models/user_repository.go

```diff
@@ -0,0 +1,37 @@
+package models
```

Comment:
That user repo is implementing the identity repository interface declared application package.